task3.4.py

```diff
@@ -0,0 +1,16 @@
+'''Напишите программу, которая будет преобразовывать десятичное 
+число в двоичное.
+Пример:
+- 45 -> 101101
+- 3 -> 11
+- 2 -> 10'''
+
+N = int(input('Введите число: '))
+N2 = str('')
+diff = 0
+
+while N > 0:
+    diff = str(N % 2)
+    N = N // 2
+    N2 += diff
```

Comment:
```suggestion
while N:
    N2 = str(N % 2) + N2 
    N = N // 2
```
И переменная `diff` не нужна.